gltf-utils/src/lib.rs

```diff
@@ -337,67 +362,42 @@ impl<T: AccessorItem> AccessorItem for [T; 4] {
     }
 }
 
-/// XYZ vertex normals of type `[f32; 3]`.
-#[derive(Clone, Debug)]
-pub struct Normals<'a>(AccessorIter<'a, [f32; 3]>);
-
 /// XYZ vertex positions of type `[f32; 3]`.
-#[derive(Clone, Debug)]
-pub struct Positions<'a>(AccessorIter<'a, [f32; 3]>);
+pub type Positions<'a> = AccessorIter<'a, [f32; 3]>;
+
+/// XYZ vertex normals of type `[f32; 3]`.
+pub type Normals<'a> = AccessorIter<'a, [f32; 3]>;
 
 /// XYZW vertex tangents of type `[f32; 4]` where the `w` component is a
 /// sign value (-1 or +1) indicating the handedness of the tangent basis.
-#[derive(Clone, Debug)]
-pub struct Tangents<'a>(AccessorIter<'a, [f32; 4]>);
+pub type Tangents<'a> = AccessorIter<'a, [f32; 3]>;
 
 /// Inverse Bind Matrices of type [[f32; 4]; 4].
-#[derive(Clone, Debug)]
-pub struct InverseBindMatrices<'a>(AccessorIter<'a, [[f32; 4]; 4]>);
+pub type InverseBindMatrices<'a> = AccessorIter<'a, [[f32; 4]; 4]>;
 
 /// Animation input sampler values of type `f32`.
-#[derive(Clone, Debug)]
-pub struct Inputs<'a>(AccessorIter<'a, f32>);
-
-/// Animation output sampler values.
-pub enum Outputs<'a> {
-    /// XYZ translations of type `[f32; 3]`.
-    Translations(AccessorIter<'a, [f32; 3]>),
-
-    /// Rotation animations.
-    Rotations(RotationsF32<'a>),
-
-    /// XYZ scales of type `[f32; 3]`.
-    Scales(AccessorIter<'a, [f32; 3]>),
-
-    /// Morph target animations.
-    Weights(MorphWeightsF32<'a>),
-}
+pub type Inputs<'a> = AccessorIter<'a, f32>;
 
 /// Vertex colors.
-#[derive(Clone, Debug)]
-enum Colors<'a> {
+#[derive(Debug, Clone)]
```

Comment:
?

Reply:
C comes before D in the alphabet.

Reply:
I must have accidentally reordered it, sorry.